lib/hydra/works/characterization/schema/base_schema.rb

```diff
@@ -10,8 +10,6 @@ class BaseSchema < ActiveTriples::Schema
     property :fits_version, predicate: RDF::Vocab::PREMIS.hasCreatingApplicationVersion
     property :exif_version, predicate: RDF::Vocab::EXIF.exifVersion
     property :original_checksum, predicate: RDF::Vocab::NFO.hashValue
-    property :mime_type, predicate: RDF::Vocab::EBUCore.hasMimeType, multiple: false do |index|
-      index.as :stored_searchable
-    end
+    property :mime_type, predicate: RDF::Vocab::EBUCore.hasMimeType
```

Comment:
@mjgiarlo the FileSet will be responsible for indexing it. Since this is Hydra::PCDM::File none of its terms are indexed.